docs/Installing.asciidoc

```diff
@@ -299,6 +299,144 @@ For older versions of openQA, you can migrate from SQLite to PostgreSQL
 according to
 <<Pitfalls.asciidoc#db-migration,DB migration from SQLite to PostgreSQL>>
 
+==== Migrate PostgreSQL database on openSUSE
+The PostgreSQL `data`-directory needs to be migrated in order to switch to a
+newer major version of PostgreSQL. The following instructions are specific to
+openSUSE's PostgreSQL and openQA packaging but with a little adaption they can
+likely be used for other setups as well. These instructions can migrate big
+databases in seconds without requiring additional disk space. However, services
+need to be stopped during the (short) migration.
+
+1. Locate the `data`-directory. Its path is configured in
+`/etc/sysconfig/postgresql` and should be `/var/lib/pgsql/data` by default. The
+paths in the next steps assume the default.
+
+2. To ease migrations, I would recommend making the `data`-directory a symlink
```

Comment:
Who is the *I* here? I suggest to write instead s/I would recommend/it is recommended/